site/src/components/Calendar.vue

```diff
@@ -325,7 +332,7 @@ $dayFontSize: 0.8em;
 .calendar-event {
   display: block;
   box-sizing: border-box;
-  border-top: 1px solid #e7e7e7 !important; //temp fix for the borders not showing
+  // border-top: 1px solid #e7e7e7 !important; //temp fix for the borders not showing
```

Comment:
Why comment out instead of keeping as-is or deleting?

Reply:
I suppose in case someone decided they wanted it back on ? I'm fine with deleting it